xdmod_data/_validator.py

```diff
@@ -272,7 +272,7 @@ def __get_dates_from_duration(duration):
         1,
     )
     if today.month < 4:
-        last_quarter_start_year = today.year - 1
+        last_quarter_start_year = today.year - 1  # pragma: no cover
     else:
         last_quarter_start_year = today.year
```

Comment:
Both this `if` and `else` will need the `pragma: no cover` because whether they are covered depends on the time of year when the test is run (e.g., we're running the tests now in June, so the `else` is not currently covered, but in January it will be the other way around).